src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -56,10 +72,37 @@ export function VideoThumbnail({
     );
   }
 
+  const progressValue = calculateProgress(videoLength, videoProgress);
+  const progressValuePercents = Math.round(progressValue * 100);
+
   const timeStampMarkup = videoLength ? (
-    <p className={styles.Timestamp}>{secondsToTimestamp(videoLength)}</p>
+    <p
+      className={classNames(
+        styles.Timestamp,
+        progressValue && styles.WithProgress,
```

Comment:
Should this be:

```suggestion
        showVideoProgress && styles.WithProgress,
```

?